lib/connection/connection_config.js

```diff
@@ -474,6 +475,14 @@ function ConnectionConfig(options, validateCredentials, qaMode, clientInfo) {
     includeRetryReason = options.includeRetryReason;
   }
 
+  let clientStoreTemporaryCredential = false;
```

Comment:
is the default correct? don't we want to use the temporal storage always? @sfc-gh-pmotacki

Reply:
On the risk assessment, they mentioned that this option is disabled by default. The customer should read the descripiton first, and need to know what the risk is.